recipes/r-gggenomes/meta.yaml

```
@@ -0,0 +1,110 @@
+{% set version = '1.0.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-gggenomes
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/gggenomes_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/gggenomes/gggenomes_{{ version }}.tar.gz
+  sha256: 3cdee64dc9156a0cb3eb5d3bb8b83df751ced28b2b84cc1d545fafda179f7538
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base >=4.2

```

Please restore to unspecified:

```suggestion
    - r-base
```